apps/web/src/pages/tenants/components/list/columns.tsx

```diff
@@ -0,0 +1,42 @@
+import { ITenantEntity } from '@novu/shared';
+import { Skeleton } from '@mantine/core';
+import { format } from 'date-fns';
+
+import { IExtendedColumn } from '../../../../design-system';
+
+const maxWidth = 60;
+
+export const columns: IExtendedColumn<ITenantEntity>[] = [
+  {
+    accessor: 'name',
+    Header: 'Name',
+  },
+  {
+    accessor: 'identifier',
+    Header: 'Tenant identifier',
+  },
```

Comment:
Let's add the withCellLoading and also an ellipsis cut for long names/identifiers with a tooltip for the full name? wdyt?

I mean like this: 
```
 Cell: withCellLoading(({ row: { original } }) => (
      <Tooltip label={original.name}>
        <Text rows={1}>{original.name}</Text>
      </Tooltip>
    )),
```

Reply:
love it! thanks for the suggetion